components/membership/membership-rest-impl/src/main/kotlin/net/corda/membership/impl/rest/v1/MemberLookupRestResourceImpl.kt

```diff
@@ -158,5 +170,18 @@ class MemberLookupRestResourceImpl @Activate constructor(
                 }
             )
         }
+
+        override fun viewGroupParameters(holdingIdentityShortHash: ShortHash): Map<String, String> {
+            val holdingIdentity = virtualNodeInfoReadService.getByHoldingIdentityShortHashOrThrow(
+                holdingIdentityShortHash
+            ) { "Could not find holding identity '$holdingIdentityShortHash' associated with member." }.holdingIdentity
+
+            return with(membershipGroupReaderProvider.getGroupReader(holdingIdentity)) {
+                groupParameters?.toMap()
+            } ?: throw ResourceNotFoundException("Could not find group parameters for holding identity " +
+                        "'$holdingIdentityShortHash'.")
```

Comment:
Nit picky and not a blocker, but it might be a bit more readable without `with`. For example:
```suggestion
            return membershipGroupReaderProvider
                .getGroupReader(holdingIdentity)
                .groupParameters
                ?.toMap() ?: throw ResourceNotFoundException("Could not find group parameters for holding identity " +
                        "'$holdingIdentityShortHash'.")
```